### PR TITLE
Promoted images should be rebuilt with merged code

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

File: config/Dockerfiles/rpmbuild/Dockerfile [hunk 1]
```diff
@@ -53,5 +53,6 @@ COPY rpmbuild-test.sh /tmp/rpmbuild-test.sh
 
 # Run the build script
 ENTRYPOINT ["bash", "/tmp/rpmbuild-test.sh"]
+#
```
Comment (johnbieren) [inline config/Dockerfiles/rpmbuild/Dockerfile]: Is this needed or a typo? Just wondering if I will need to add this to future containers, not trying to nit pick

Comment (scoheb) [reply config/Dockerfiles/rpmbuild/Dockerfile]: this is a dummy change I need to validate the change I am doing surrounding the promotion of images AFTER a merge

**Comment by scoheb** [comment]
[test] segfault seems in logs

**Comment by scoheb** [comment]
[test]

**Comment by scoheb** [comment]
[merge]

**Pull request merged**